docs/Encoder.md

```diff
@@ -1,57 +1,81 @@
-### 2. D-Sub Connector
-Each d-sub connecter has 9 ports – 6 for differential ABZ signals, 2 for ground signals and 1 for 5 V supply which can be connected to encoder as a DC supply. A jumper should be connected if encoder is powered by 5 V from AMDC. More information on d-sub connector can be found [here]( https://www.alliedelec.com/m/d/c3366066d9274ddf3c20bc3008518f5b.pdf?src-supplier=Allied+Electronics)
-### 3. Differential to single-ended converters
-After passing through d-sub connector, differential ABZ signals (A, A̅, A, B̅, Z, Z̅) are converted to single-ended ABZ signals (OUT A, OUT B, OUT Z) using [AM26C32](http://www.ti.com/lit/ds/symlink/am26c32.pdf) chip. This chip is supplied by Vcc = 5 V DC source from AMDC. The following table summarizes absolute maximum ratings taken from the datasheet:   
 
-| Parameter | MIN            | MAX                | 
-|-------------------|-----------------|----------------------|
-| Supply voltage Vcc             |                 | 7 V          |
-| Input voltage (A, A̅, A, B̅, Z or Z̅)           | -11 V            | 14 V     |
-| Differential input voltage      | -14 V            | 14 V    |
-| Output voltage (OUT A, OUT B, OUT Z) | -0.5 V              | (Vcc + 0.5) V  |
+The AMDC REV D hardware supports two independent encoder interfaces. These must be incremental encoders outputting differential ABZ signals (i.e. the encoder outputs both the non-inverting and inverting signal, e.g. A and A̅). If the encoder only provides single-ended ABZ outputs, an adapter must be used between the encoder and the AMDC which converts the single-ended outputs to the differential signals required by the AMDC. For proper operation, the encoder ground should be connected to the AMDC ground via a pin on the DB9 connector. For powering the encoder, the user has two options: the AMDC provides 5 V power via the DB9 connector, or the user can use an external power supply.
 
-Exposure to absolute maximum ratings for extended period of time may affect device reliability. Therefore, it is recommended to operate under recommended operating conditions:
+### 2. DB9 Connector
 
-| Parameter | MIN            | NOM                | MAX|
-|-------------------|-----------------|----------------------|----------------------|
-| Supply voltage Vcc             | 4.5 V | 5 V          | 5.5 V |
-| High-level input voltage (logic HIGH) | 2 V |      | Vcc     |
-| Low-level input voltage (logic LOW)   | 0 V |      | 0.8 V    |
-| Common-mode input voltage | -7 V      |      | 7 V  |
+Each D-sub connecter has 9 pins – 6 for differential ABZ signals, 2 for ground signals and 1 for 5V supply to encoder. A jumper should be installed if the encoder is powered by 5 V from AMDC. There are two jumpers numbered on the silkscreen as JP2 and JP3 (each for one encoder input). 

```

Comment:
How do we know which jumper is for which encoder? For that matter, how do we know which encoder on the connector is `Encoder 1` (top or bottom?) and which is `Encoder 2` (top or bottom?) from your block diagram?

Reply:
```suggestion
Each D-sub connecter has 9 pins – 6 for differential ABZ signals, 2 for ground signals and 1 for 5V supply to encoder. A jumper should be installed if the encoder is powered by 5 V from AMDC. There are two jumpers numbered on the silkscreen as JP2 for Encoder 1 and JP3 for Encoder 2, as shown in the block diagram. Encoder 1 and Encoder 2 correspond to top and bottom encoders in the picture below.

```